app/webpacker/components/RegistrationsV2/RegistrationAdministration/RegistrationActions.jsx

```diff
@@ -43,7 +43,10 @@ function V3csvExport(selected, registrations, competition) {
         registration.competing.registration_status
       },${
         registration.registrant_id
+      },${
+        registration.competing.waiting_list_position || null
```

Comment:
Isn't the waiting list position just "null or integer" anyways?
Please let me know if I'm missing something, but this reads as "fall back to null if it's null" to me

Reply:
It comes out as `undefined` without the `|| null`. So not much of a difference either way, `null` just feels a bit better

Reply:
Then just fall back to an empty string